dandiapi/api/migrations/0001_stagingapplication.py

```diff
@@ -5,19 +5,11 @@
 import django.db.models.deletion
 import oauth2_provider.generators
 
-# This migration is needed to support wildcards in OAuth redirect URIs in staging.
-# The model created here is only actually used in staging.
-
 
 class Migration(migrations.Migration):
     # This has no dependencies, but isn't considered "initial" by Django's definition
     initial = False
 
-    # This is creating a swappable model, so it needs to exist before the OAuth2 models are created
-    run_before = [
```

Comment:
This model doesn't matter anymore, so there's no need to keep a particular ordering with respect to OAuth-Toolkit.